templates/workspace_services/guacamole/user_resources/README.md

```diff
@@ -0,0 +1,75 @@
+# Guacamole User Resources
+
+This folder contains user resources that can be deployed with the Guacamole workspace service:
+
+- linuxvm - a Linux-based virtual machine (expects an Ubuntu 18.04-based VM)
+- windowsvm - A Windows-based virtual machine
+
+
+## Customising the user resources
+
+The `guacamole-azure-linuxvm` and `guacamole-azure-windowsvm` folders follow a consistent layout.
+To create a template To update one of these templates (or to create a new template based on these folders) to use different image details or VM sizes, there are a few files that need to be updated:
```

Comment:
Typo: 
To create a template To update one of these templates

Reply:
Oops - have fixed the broken sentence